include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -27,7 +27,8 @@ def InstanceOp : FIRRTLOp<"instance", [
                        APIntAttr:$portDirections, StrArrayAttr:$portNames,
                        AnnotationArrayAttr:$annotations,
                        PortAnnotationsAttr:$portAnnotations,
-                       BoolAttr:$lowerToBind);
+                       BoolAttr:$lowerToBind,
+                       OptionalAttr<SymbolNameAttr>:$inner_sym);
```

Comment:
Modules will still have `Symbol` instead of `inner_sym` ?

Reply:
Yeah exactly, because the modules are globally visible. Resolving the inner names requires the module symbol, plus whatever inside it you're referring to.